mindmaps-test/src/test/java/io/mindmaps/graql/internal/analytics/ScalingTestIT.java

```diff
@@ -78,7 +82,7 @@ public void setUp() throws InterruptedException {
     }
 
     @Test
-    public void testAll() throws InterruptedException, ExecutionException {
+    public void testAll() throws InterruptedException, ExecutionException, MindmapsValidationException {
```

Comment:
At a later date when analytics stabliseses you should put in some assertions in these tests. It will make them maintainable and useful in the future.